systemvm/debian/opt/cloud/bin/configure.py

```diff
@@ -1002,7 +1002,7 @@ def configure_l2tpIpsec(self, left, obj):
 
         secret = CsFile(vpnsecretfilte)
         secret.empty()
-        secret.addeq("%s : PSK \"%s\"" % (left, psk))
+        secret.addeq(": PSK \"%s\"" % (psk))
```

Comment:
@GutoVeronezi LGTM, but should we revert to old ? I don't know what the syntax really means 
```        secret.addeq("%s %%any : PSK \"%s\"" % (left, psk))
```

Reply:
Update, from https://github.com/apache/cloudstack/issues/4281#issue-684586236 it seems the user's fix correlates with the syntax used.